catalog-info.yaml

```
@@ -310,9 +309,9 @@ spec:
       name: beats-libbeat
       description: "Beats libbeat pipeline"
     spec:
-#      branch_configuration: "main 7.17 8.* v7.17 v8.*" TODO: temporarily commented to build PRs from forks
+      branch_configuration: "main 7.* 8.* v7.* v8.*"
```

I'm not sure if tags should trigger any build, they are in fact managed by the Unified Release process and hence tag from commits that have been already tested it out.

```suggestion
      branch_configuration: "main 7.17 8.*
```